src/commands/Minion/slayertask.ts

```diff
@@ -32,13 +34,42 @@ export default class extends BotCommand {
 		});
 	}
 
+	public getAlternateMonsterList(currentTask: SlayerTaskTable, assignedTask: AssignableSlayerTask | null) {
+		if (currentTask && assignedTask) {
+			const altMobs = assignedTask.monsters;
+			const alternateMonsters = killableMonsters
+				.filter(m => {
+					return altMobs.includes(m.id) && m!.id !== assignedTask.monster.id;
+				})
+				.map(m => {
+					return m!.name;
+				});
+			return alternateMonsters.length > 0 ? ` (${alternateMonsters.join('/')})` : '';
+		}
+		return '';
+	}
+
+	public async returnSuccess(msg: KlasaMessage, message: string, autoslay: boolean) {
+		if (autoslay) {
+			// await msg.channel.send(message);
+			try {
+				return this.client.commands.get('autoslay')!.run(msg, ['']);
+			} catch (e) {
+				return msg.channel.send('It was not possible to auto-slay this task. Please, try again.');
```

Comment:
Remove comma from, 'Please, try again'

Reply:
I want to give emphasis on try again, hence the comma.